src/components/pages/editor/editor-contact.tsx

```diff
@@ -0,0 +1,45 @@
+import Link from 'next/link'
+
+import { teamData } from './editor-team'
+
+type EditorContactProps =
+  | { firstName: string; lastName?: string }
+  | { firstName?: string; lastName: string }
+
+export function EditorContact({ firstName, lastName }: EditorContactProps) {
```

Comment:
I would define it as

```typescript
function EditorContact(contact: TeamDataEntry) {

}
```

In the editor you can call it with the editor contacts and on metadata with my contact information. Then you do not need `isEditorContactPerson`.